src/library/basetracktablemodel.cpp

```diff
@@ -872,6 +858,23 @@ QVariant BaseTrackTableModel::roleValue(
             // Undecidable
             return Qt::PartiallyChecked;
         }
+        break;
+    }
+    // Right align BPM, duration and bitrate so big/small values can easily be
+    // spotted by length (number of digits)
+    case Qt::TextAlignmentRole: {
+        switch (field) {
+        case ColumnCache::COLUMN_LIBRARYTABLE_BPM:
+        case ColumnCache::COLUMN_LIBRARYTABLE_DURATION:
+        case ColumnCache::COLUMN_LIBRARYTABLE_BITRATE: {
+            // We need to cast to int due to a bug similar to
+            // https://bugreports.qt.io/browse/QTBUG-67582
+            return static_cast<int>(Qt::AlignVCenter | Qt::AlignRight);
+        }
+        default:
+            return QVariant(); // default AlignLeft for all other columns
+        }
+        break;
```

Comment:
why break here?

Reply:
I added the break just for consistency with the other role case statements, and as a personal preference I always explicitly add a break at the end of a case in a switch statement. I agree it will never reach that break since there are returns prior to it.

Reply:
Yes, other role cases' default case would break to return the raw role value.
Though, IMO the break is obsolete for `Qt::CheckStateRole` and `Qt::TextAlignmentRole`.

Reply:
I agree the breaks are unnecessary for the two roles you mention. I'm fine reverting my additions of the breaks.

Reply:
Great. This is just a small change, so please squash the commits into one and force-push.

Reply:
Sorry, I don't know how to do that.

Reply:
https://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

afterwards `git push --force` otherwise github will refuse the new commits since you rewrote the history.